web/pdf_viewer.js

```diff
@@ -203,6 +203,8 @@ class PDFViewer {
 
   #altTextManager = null;
 
+  #toastManager = null;
```

Comment:
Here, and in lots of other spots in the patch, please place class fields in *alphabetical* order.